validate.lic

```diff
@@ -471,7 +471,13 @@ class DRYamlValidator
 
     warn('***YOU HAVE OUTDATED SETTINGS*** THE SETTING predict_event IS NO LONGER USED. READ MORE HERE: https://elanthipedia.play.net/Lich_script_repository#astrology')
   end
-
+  
+  def assert_that_moons_is_deprecated(settings)
+     return unless settings.astrology['moons']
```

Comment:
Should this be `settings.astrology_training`?

Reply:
And since its a list/array, it should use `include?`
```
2.0.0-p647 :001 > test = ['hey']                                            
 => ["hey"]                                                                 
2.0.0-p647 :002 > test['hey']                                               
TypeError: no implicit conversion of String into Integer                    
        from (irb):2:in `[]'                                                
        from (irb):2                                                        
        from /home/jonas/.rvm/rubies/ruby-2.0.0-p647/bin/irb:12:in `<main>' 
2.0.0-p647 :003 > test.include?('hey')                                      
 => true                                                                    
2.0.0-p647 :004 >                                                           
```